apps/point_cloud_editor/src/cloud.cpp

```diff
@@ -109,12 +109,11 @@ Cloud::Cloud (const Cloud &copy)
   select_translate_y_(copy.select_translate_y_),
   select_translate_z_(copy.select_translate_z_)
 {
-  std::copy(copy.center_xyz_, copy.center_xyz_+XYZ_SIZE, center_xyz_);
-  std::copy(copy.cloud_matrix_, copy.cloud_matrix_+MATRIX_SIZE, cloud_matrix_);
-  std::copy(copy.select_matrix_, copy.select_matrix_+MATRIX_SIZE,
-            select_matrix_);
-  std::copy(copy.color_, copy.color_+RGB, color_);
-  std::copy(copy.highlight_color_, copy.highlight_color_+RGB, highlight_color_);
+  std::copy_n(copy.center_xyz_, XYZ_SIZE, center_xyz_);
```

Comment:
Could you explain a bit why you changed `std::copy` to `std::copy_n` (this instance and the others)? `std::copy_n` might be slightly more expressive ("copy this many values from here to there" instead of "copy values from this range to there"), but there is nothing bug-prone about `std::copy`, right? And `std::copy` might be faster because it [might use std::memmove if it's safe to do so](https://en.cppreference.com/w/cpp/algorithm/copy) (copy_n's documentation doesn't say that, possibly because it is less strict with overlapping regions)

Reply:
It was @kunaltyagi who asked, but I've changed back to `std::copy` for the advantages you outlined.